PURL-TYPES.rst

```diff
@@ -221,6 +207,34 @@ nuget
 
       pkg:nuget/EnterpriseLibrary.Common@6.0.1304
 
+oci-artifact
+------------
+``oci-artifact`` for all artifacts stored in OCI Distribution-spec conformant
+registries including container images built by docker and others:
```

Comment:
```suggestion
registries including container images built by Docker and others:
```

Reply:
Just for context, Docker is used to refer to **Docker Inc**, while docker is used to refer to the `docker` cli. When referring to process containers, we've tried to shift to _container images_ to be vendor neutral.